packages/fleet/lib/utils/errors.ts

```diff
@@ -0,0 +1,23 @@
+type Jsonable = string | number | boolean | null | undefined | readonly Jsonable[] | { readonly [key: string]: Jsonable } | { toJSON(): Jsonable };
```

Comment:
could be type-fest or could be stored globally in types/utils

Reply:
didn't want to bring type-fest just for that. Where in types/utils would you put it?

Reply:
packages/types/lib/utils.ts